overview.tex

```diff
@@ -1,25 +1,31 @@
 \subsection*{Introduction}
 
-    The Performance Profile of a solver is the cumulative distribution
-    function of a performance metric, e.g., CPU time, number of functions
-    evaluations, number of iterations, or others, that we will call \emph{cost}.
-
-    It was first described by \textcite{Dolan:2002du} and is presented as a 
-    graphic that shows the relative performance profile of different solvers, 
-    according to a chosen metric, for a given set of problem instances.
-
-    Given a set $\Pset$ of problems and a set $\Sset$ of solvers, for each 
-    problem $p \in \Pset$ and solver $s \in \Sset$, we define $t_{ps}$ as the  
-    cost required to solve problem $p$ by solver $s$ and
-    \begin{align*}
-      r_{ps} = \frac{t_{ps}}{\min\{t_{ps}: s \in \Sset\}}
-    \end{align*}
-    as the performance ratio of solver $s$ for the problem $p$ when compared
-    with the best performance by any solver on this problem.
-
-    \citeauthor{Dolan:2002du} define the probability of a solver $s \in \Sset$ to
-    solve one problem within a factor $\tau \in \mathds{R}$ of the best
-    performance ratio as the function
+    When running an optimization solver on a set  $\Pset$ of problems, it is
+    usual to record   information of interest --- such as CPU time, number of functions     evaluations, number of iterations, or others --- which we will reference    as \emph{cost}. This process is  called Benchmark. 
+
+    Benchmarking an optimization software has been a necessity and some works 
+    has showed that uncovered deficiencies in the software and has 
+    generally led to software improvements~\cite{url:mittelmann,Mittelmann:1999fb,Dolan:2006kl}. In this
+    sense,
+    \textcite
+    {Dolan:2002du} developed a tool to compare optimization solvers benchmarks:
+    the Performance Profile.
+ 
+
+    The Performance Profile is a means 
+    to evaluate and compare the performance of a set of solvers $\Sset$ on a 
+    test set $\Pset$. It is presented as a graphic that shows the cumulative
+    distribution function of different solvers performances,
+    according to the  chosen cost metric, for a given set of problem instances.
+
+    For each problem $p \in \Pset$ and solver $s \in \Sset$, let $t_{ps}$
+    be the cost required to solve problem $p$ by solver $s$ and
+    \begin{align*} r_{ps} = \frac{t_{ps}}{\min\{t_{ps}: s \in \Sset\}}
+    \end{align*} be the performance ratio of solver $s$ for the problem $p$
+    when compared with the best performance by any solver on this problem.
```

Comment:
Have you used some sort of automatic line breaker? This align shouldn' t be written like this.